<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2590</width>
    <height>1950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <widget class="QTextEdit" name="CurSysSpecial"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="SessionSystemVisits">
      <property name="text">
       <string>Session systems: xxx</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SystemsVisited">
      <property name="text">
       <string>Systems visited: xxx</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalSystemsVisited">
      <property name="text">
       <string>totalSystemsVisited</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SystemName">
      <property name="text">
       <string>System: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="StationName">
      <property name="text">
       <string>Station: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="JumpDistanceLast">
      <property name="text">
       <string>Last jump distance: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="JumpDistanceRecords">
      <property name="text">
       <string>Jump distance shortest: -, longest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="FuelScoopedTotal">
      <property name="text">
       <string>Fuel scooped: 0, lowest fuel level reached: 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SessionScans">
      <property name="text">
       <string>Session Scans: xxx</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SessionPlanets">
      <property name="text">
       <string>Session Planet: xxx</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SessionSpecialPlanets">
      <property name="text">
       <string>Session Ammonia / Earthlike / Water Worlds</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SessionStations">
      <property name="text">
       <string>Session Stations Visited: xxx</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Deaths">
      <property name="text">
       <string>CMDR Deaths: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="StarMassRecords">
      <property name="text">
       <string>Star mass  lowest: -, highest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="StarRadiusRecords">
      <property name="text">
       <string>Star radius smallest: -, largest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="DistanceLSRecords">
      <property name="text">
       <string>Planet Light Seconds from star closest: -, furthest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PlanetRadiusRecords">
      <property name="text">
       <string>Planet radius smallest: -, largest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PlanetGravityRecords">
      <property name="text">
       <string>Planet gravity lowest: -, highest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LandableRadiusRecords">
      <property name="text">
       <string>Landable Planet Smallest: -, largest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LandableGravityRecords">
      <property name="text">
       <string>Landable gravity lowest: -, highest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Age_MYRecords">
      <property name="text">
       <string>Star youngest: -, oldest: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TradingStats">
      <property name="text">
       <string>Session: last trade -Cr, total trades -Cr</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="SysNamAutoClipboard">
      <property name="text">
       <string>System name auto clipboard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="IgnoreTraining">
      <property name="text">
       <string>IgnoreTraining</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>UTC time to clipboard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Station to clipboard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>System to clipboard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_UTCArrivedAtSystem">
      <property name="text">
       <string>UTC arrived at SYSTEM to clipboard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_CreditsToClipboard">
      <property name="text">
       <string>Credits to clipboard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_TD_SystemStation">
      <property name="text">
       <string>TradeDangerous SYSTEM/STATION</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2590</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionDiscord_Paste"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDiscord_Paste">
   <property name="text">
    <string>Discord Paste</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
